### PR TITLE
Program to explain functions in JS

### LINKED ISSUE #30: Write a program to explain the use of functions?

### PR DESCRIPTION
**Pull request by shubham0109**
This pull request fixes #30

### CODE REVIEW

File: functions in JS/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<html>
```
Comment (t2013anurag) [inline functions in JS/index.html]: move the file to functions directory, and rename the file to functions_in_js.html.

File: functions in JS/main.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+
```
Comment (t2013anurag) [inline functions in JS/main.js]: move the file to functions directory, and rename the file to functions_in_js_main.html.

File: functions in JS/main.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+convertToRoubles(dollars);
+
+function convertToINR (dollars){
+	var dollars = 30;            // Here the var dollars has a local scope i.e the var dollars with value 30 can 
+	return dollars*60;			 // only be used inside the function convertToINR.	
+}
+var rupees = convertToINR(dollars); // Here the dollars refer to the global dollar variable, although inside the function
+									// it is changed locally
+					    			// outside the val of dollars is still 15
```
Comment (t2013anurag) [inline functions in JS/main.js]: add a new line,

File: functions in JS/index.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+<body>
+	<h1>
+		FUNCTIONS IN JS
+	</h1>
+
+	
+</body>
+
+</html>
```
Comment (t2013anurag) [inline functions in JS/index.html]: add a new line.

File: functions in JS/index.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    <meta charset="UTF-8">
+    
+    <script src="main.js"></script>
+
+</head>
+
+<body>
+	<h1>
+		FUNCTIONS IN JS
+	</h1>
```
Comment (t2013anurag) [inline functions in JS/index.html]: remove all extra lines from the code

File: functions in JS/index.html (hunk 1 above)
Comment (shubham0109) [reply functions in JS/index.html]: @t2013anurag  I guess there's no functions directory right now. Do you want me to create a functions directory and push the JS file into it?

File: functions in JS/index.html (hunk 5 above)
Comment (shubham0109) [reply functions in JS/index.html]: @t2013anurag Here are you suggesting to remove all the white spaces or should I shorten my code?

Comment (t2013anurag) [reply functions in JS/index.html]: remove extra new lines., use only one return statement in code.

File: functions in JS/index.html (hunk 1 above)
Comment (t2013anurag) [reply functions in JS/index.html]: @shubham0109 - yes you can create one,

**Pull request merged**